### DISCUSSION TITLE
스터디 - 3주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#29] [General]
> 3주차 스터디 목록입니다. 스터디에 주제와 갯수는 강제가 아닌 자율적으로 속도 조절해서 해주시면 됩니다.

## 스터디 일정
2021.11.25(목) 
21:00 ~ 21:15 (팀원들 PR 남긴 거 리뷰 시간)
21:15 ~ 21:35 (1:다)
21:35 ~ 21:50 (모의 면접 후 모여서 마무리로 회고록 작성 후 해산)

## 스터디 모음
영생 
--
- CSS - 박스 모델(Box-model)
- FrontEnd - 웹팩(Webpack)

동기
---
- JavaScript - 이벤트 루프(Event-Loop)
- JavaScript - 표현식과 문(Expression And Statement)

남은
--
- JavaScript - require vs import
- JavaScript - promise, promiseAll, promiseAllSettled, promiseRace, promiseAny (2주차 기술면접  Promise와 Callback  - 이어서)

송현
--  
- JavaScript - 이벤트 버블링과 위임
- Network - REST-API

두현
- React - React.memo, useCallback
- React - 클래스 컴포넌트(Class-based Components)

수민 
- FrontEnd - CORS

**Comment by vi2920va** [comment]
송현 - 3주차 스터디 주제
- JavaScript - 이벤트 버블링과 캡쳐 위임
- Network - REST-API

**Comment by ki7348** [comment]
동기 - 3주차 스터디 주제
* JavaScript - 이벤트 루프
* JavaScript -  표현식과 문

**Comment by whdudtod1273** [comment]
영생 - 3주차 스터디 주제

- CSS - box-model
- FrontEnd - webpack
- Network - HTTP 상태코드

**Comment by NAMU1105** [comment]
남은 - 3주차 스터디 주제

- JavaScript - require vs import
- JavaScript - promise, promiseAll, promiseAllSettled, promiseRace, promiseAny

금주 스터디는 개인 사정으로 불참하게 되었습니다.

**Reply by vi2920va** [reply]
- 남은님 스터디 진행하는 주제가 어떤 것에 속하는지 대분류-중분류로 나누어주세요
- 그리고 JavaScript Promise 말하시는거면 제가 정리한 거에서 보태서 해야될거 같아요

**Reply by NAMU1105** [reply]
> * 남은님 스터디 진행하는 주제가 어떤 것에 속하는지 대분류-중분류로 나누어주세요
> * 그리고 JavaScript Promise 말하시는거면 제가 정리한 거에서 보태서 해야될거 같아요

지난 주제들 모아서 볼 수 있는 메뉴가 하나 있었으면 좋겠습니다~

**Reply by vi2920va** [reply]
> 남은님 스터디 진행하는 주제가 어떤 것에 속하는지 대분류-중분류로 나누어주세요
> 
> 그리고 JavaScript Promise 말하시는거면 제가 정리한 거에서 보태서 해야될거 같아요
> 
> 
> 
> 지난 주제들 모아서 볼 수 있는 메뉴가 하나 있었으면 좋겠습니다.

따로 보는 건 전주차 스터디 목록이나 폴더에 리드미 파일에 목록이 있습니다

**Reply by NAMU1105** [reply]
매주매주 리스트를 확인하면서 다뤘던 주제인지 봐야하는 번거로움이 있을 것 같아서요~

다른 분들에게도 여쭤보고 괜찮으면 전체 리스트 만들면 좋을 것 같습니다.

**Comment by Doodream** [comment]
두현 - 3주차 스터디 주제
- React - React.memo, useCallback
- React - Class-based Components

**Reply by vi2920va** [reply]
두현님  React.meno와 useMemo와 같은거 맞나요?

**Reply by Doodream** [reply]
아니요 달라요;  useMemo는 시간이 되면 연결해서 주제에 넣으려고 했어요

**Reply by Doodream** [reply]
React - React.memo, useCallback  제목을 
React - React Operation && Optimization Techniques로 수정합니다.

**Reply by vi2920va** [reply]
두현님 파일명 제목 바꿔서 올려주세요!

**Comment by ttumzzi** [comment]
수민 - 3주차 스터디 주제

- Frontend - CORS

**Comment by vi2920va** [comment]
요번주에 어땟는지 10점만점에 점수로 표현해주세요(회고록)

**Reply by vi2920va** [reply]
송현 : 5점입니다.  요번주 프로젝트 진행과 힘들었습니다

**Reply by whdudtod1273** [reply]
영생 : 8점

**Reply by Doodream** [reply]
두현 : 비교 할 수 있는 경우가 없어서 저는 8점 으로 하겠습니다. 다양한 분에게 질문을 받고 부족한 점을 찾을 수 있으면 좋겠습니다.

**Reply by ttumzzi** [reply]
수민: 8점. CORS가 좀 두루뭉실하게만 이해되어서 쉽지 않았습니다. 더 자세히 적어서 다음주에 추가하겠습니다.

**Reply by ki7348** [reply]
동기: 8점입니다. 마이크로태스크큐에 대해서 추가로 정리하겠습니다.

**Reply by NAMU1105** [reply]
남은: 6점입니다. 준비시간이 너무 부족했습다. 다음 주부터는 더 준비 많이 하겠습니다.
